src/lib/snarky/src/camlsnark_c/camlsnark_linker_flags_gen.ml

```diff
@@ -9,6 +9,7 @@ let () =
     ( match l with
     | "Darwin" ->
         [ sprintf "-Wl,-force_load,%s/libcamlsnark_c_stubs.a" cwd
+        ; "-L/usr/local/Cellar/openssl/1.0.2q/lib"
```

Comment:
We’d be more robust against OpenSSL upgrades if you used the /opt synlink here (assuming it’s okay to use a symlink in the linker flags)

Reply:
done